keps/sig-node/3327-align-by-socket/kep.yaml

```diff
@@ -28,13 +28,15 @@ latest-milestone: "v1.25"
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.25"
-  beta: "v1.26"
-  stable: "v1.27"
+  beta: "v1.27"
+  stable: "v1.28"
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
   - name: "CPUManagerPolicyAlphaOptions"
+  - name: "CPUManagerPolicyBetaOptions"
+  - name: "CPUManagerPolicyOptions"
```

Comment:
It's odd to have 3 separate feature gates for a feature - is that right?

Reply:
@sftim  Please note that `align-by-socket` option proposed here customizes the default behavior of CPU Manager static policy. There are other possible options like `full-pcpus-only`, `distribute-cpus-across-numa` that can be passed to the feature flag (`--cpu-manager-policy-options`). Since, we wanted to avoid proliferation of feature gates we had to deviate from the standard pattern of a feature gate per policy option and identified three feature gates that covered to cover already existing options or the ones proposed in the future:

1. `CPUManagerPolicyOptions`: for controlling the ability to customize the CPU Manager static policy behavior.
2. `CPUManagerPolicyAlphaOptions` for controlling visibility of alpha quality options.
3. `CPUManagerPolicyBetaOptions` for controlling visibility of Beta quality options.

Please refer to the google groups discussion [here](https://groups.google.com/u/0/g/kubernetes-sig-architecture/c/Nxsc7pfe5rw) for more details on why we needed these three feature gates.